addOns/ascanrulesBeta/src/test/java/org/zaproxy/zap/extension/ascanrulesBeta/EnvFileScanRuleUnitTest.java

```diff
@@ -63,7 +72,26 @@ public void shouldAlertWhenRequestIsSuccessful() throws Exception {
                 createHandler(
                         path,
                         newFixedLengthResponse(
-                                Response.Status.OK, NanoHTTPD.MIME_HTML, RELEVANT_BODY)));
+                                Response.Status.OK, "application/octet-stream", RELEVANT_BODY)));
+        HttpMessage msg = this.getHttpMessage(path);
+        rule.init(msg, this.parent);
+        // When
+        rule.scan();
+        // Then
+        assertThat(alertsRaised, hasSize(1));
+        assertEquals(1, httpMessagesSent.size());
+        Alert alert = alertsRaised.get(0);
+        assertEquals(Alert.RISK_MEDIUM, alert.getRisk());
+        assertEquals(Alert.CONFIDENCE_MEDIUM, alert.getConfidence());
+    }
+
+    @Test
+    public void shouldAlertWhenRequestIsSuccessfulNoContentType() throws Exception {
```

Comment:
- The public modifier is not needed.
 - This should be merged with the above and be parameterised on the Content-Type (both are not testing just the status code but also the Content-Type as well,  e.g. `shouldAlertWhenRequestIsSuccessfulWithExpectedContentType`). Although this would require not extend `AbstractAppFilePluginUnitTest` or override `shouldAlertWhenRequestIsSuccessful` to do nothing.

Reply:
fixed (I think) this would be my first try with parameterized and overriding to do nothing